index.js

```diff
@@ -38,20 +38,42 @@ type QuestionPage implements Page {
     sectionId: Int!
 }
 
-type Answer {
-    id: Int
+interface Answer {
+    id: Int!
+    description: String
+    guidance: String
+    qCode: String
+    label: String
+    type: AnswerType!
+    mandatory: Boolean
+    questionPageId: Int
+}
+
+type BasicAnswer implements Answer {
+    id: Int!
     description: String
     guidance: String
     qCode: String
     label: String
-    type: AnswerType
+    type: AnswerType!
+    mandatory: Boolean
+    questionPageId: Int
```

Comment:
@samiwel could also make a change so that `questionPageId` and `answerId` are required? They should never be orphaned, so it makes sense for them to be required